x-pack/platform/plugins/shared/stack_connectors/server/connector_types/email/index.ts

```diff
@@ -170,6 +170,16 @@ const ParamsSchemaProps = {
       }),
     }),
   }),
+  attachments: schema.maybe(
+    schema.arrayOf(
+      schema.object({
+        content: schema.string(),
```

Comment:
Can you ask Tim how large the largest reports he's seen are?  I would guess CSV reports, since there are folks who basically export LARGE amounts of data this way.  The implication being we may need to stream the output rather than have it all in memory at once.  So we will likely want to set some kind of limit for now, maybe like 20mb or so, in the follow-on PR where we actively use this.